### PR TITLE
Add integration tests

### PR DESCRIPTION
**Pull request by flavio**
This makes possible to run the integration tests previously defined inside of our internal CI infrastructure.

### CODE REVIEW

**Comment by flavio** [comment]
We could start these tests on travis, we only need to force the usage of the "old" execution mode (VM) instead of the new one (docker container).

**Comment by flavio** [comment]
BTW, the "docker images" integration tests are missing, I ported them by they are not working. I'll submit another PR

File: spec/helper.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
+require "cheetah"
+require "pathname"
+
+class Settings
+  VULNERABLE_IMAGE_REPO = "zypper-docker-tests-vulnerable-image"
+  VULNERABLE_IMAGE_TAG = "0.1"
+  VULNERABLE_IMAGE = "#{VULNERABLE_IMAGE_REPO}:#{VULNERABLE_IMAGE_TAG}"
+
```
Comment (mssola) [inline spec/helper.rb]: empty line

**Comment by mssola** [comment]
LGTM :clap: We can fix that empty line later on, since we're still building this up :)

**Pull request merged**